### DISCUSSION TITLE
Support looting/fortune?

### DISCUSSION THREAD

**Discussion by Aceplante** [vectorwing/FarmersDelight#865] [Ideas]
### Minecraft version

1.18.2

### Farmer's Delight version

1.2.3

### Forge version

40.2.9

### Description

I would like to suggest drops are able to be effected by certain enchanments namely looting/fortune or others we define in a json file. You dont need to use this functionality in any recipes by default, but it would be nice to have so we can add it ourselves.

I imagine this would work something to the effect of the following:
"tool": {
    "tag": "forge:tools/knives"
  },
          "functions": [
            {
              "function": "minecraft:looting_enchant",
              "count": 1
            }
          ],
                "enchantments": [
                  {
                    "enchantment": "minecraft:fire_aspect",
                    "levels": {
                      "min": 1
                    }
                  }

**Comment by vectorwing** [comment]
I'm not sure what you mean here. Looting and Fortune in what, exactly?

Some FD blocks already react to Fortune where applicable, as they're based on vanilla loot tables (crops, for example). The Cutting Board is also able to react to Fortune by increasing the odds of chance drops.

**Reply by Aceplante** [reply]
Ah, I did not know that fortune worked on cutting boards, but I dont think knives accept fortune, only looting, which also doesnt work with collecting straw from grass.

As for fire aspect, I think I also meant on the cutting board, so maybe dropping cooked food when cut with a fire aspect knife for example. I dont expect you to allow this by default, but I would like to be able to add checks like that myself.

**Comment by ada-fruit** [comment]
I enchanted a diamond knife with Fortune 2 via an anvil and a book. It gave me 64 straw and 64 rice from 64 rice panicles. It gave me 32 cabbage leaves from 16 cabbage. So it seems like it's not supported 🤷‍♀️

I'm playing 1.21.1 on Fabric, with Farmer's Delight, Cooking for Blockheads, JEI, and [Vegan Delight](https://www.curseforge.com/minecraft/mc-mods/vegan-delight) installed, on a server with all of those except JEI installed.

**Reply by RaymondBlaze** [reply]
The Fortune only affects chance in Cutting Board, your mentioned example is all about results that is already of chance of 100%, of course Fortune would not make a difference.
